packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import Color from "@khanacademy/wonder-blocks-color";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
+
+export default class TextField extends React.Component<{||}> {
+    render(): React.Node {
+        return (
+            <input
+                className={css([
+                    styles.input,
+                    typographyStyles.LabelMedium,
+                    styles.default,
+                ])}
+                placeholder="Placeholder"
+            />
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    input: {
+        width: "100%",
+        height: 40,
+        borderRadius: 4,
+        boxSizing: "border-box",
+        paddingLeft: Spacing.medium_16,
+        margin: 0,
+        outline: "none",
+        boxShadow: "none",
+        "::placeholder": {
+            color: Color.offBlack64,
+        },
+    },
+    default: {
+        background: Color.white,
+        border: `1px solid ${Color.offBlack16}`,
+        color: Color.offBlack,
+    },
```

Comment:
It's really nice how you splitted core CSS and theme here. This will make it easier to override with `error` and `disabled` once you include more props :)